pkg/reconciler/common/releases.go

```diff
@@ -88,7 +88,7 @@ func ComponentDir(instance v1alpha1.TektonComponent) string {
 	case *v1alpha1.TektonConfig:
 		return filepath.Join(koDataDir, "tekton-config")
 	case *v1alpha1.TektonResult:
-		return filepath.Join(koDataDir, "tekton-result")
+		return filepath.Join(koDataDir, "tekton-results")
```

Comment:
may be we can keep `result` here 😅  similar to others
and add a condition in script similar to what we have for triggers
```
   if [[ $comp == "triggers" ]]; then
     dir="trigger"
    fi
```

Reply:
i thought about it, but i think this will be better as the script will be a bit more simple.